src/manual/task-execution/provisioning.md

```diff
@@ -0,0 +1,46 @@
+---
+filename: task-execution/provisioning.md
+title: Provisioning
+order: 40
+---
+
+In the simplest case, a worker implementation is installed and configured
+manually on a host and executes tasks indefinitely. For most purposes, it is
+far more cost-effective to dynamically adjust the pool of available workers
+based on the work to be done. This process is called provisioning.
```

Comment:
This process is called provisioning -> This process entails both provisioning new instances (often referred to simply as "provisioning") and terminating running instances (typically a function of the worker itself).